Cargo.toml

```
@@ -59,5 +59,10 @@ assets = [
   ["assets/completions/_zellij", "usr/share/zsh/vendor-completions/_zellij", "644"],
 ]
 
+[package.metadata.binstall]
+pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.{ archive-format }"
```

According to [this comment](https://github.com/ryankurte/cargo-binstall/issues/38#issue-823602518), I would say the `archive-format` should be `tar.gz` :)

```suggestion
pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.tar.gz"
```